src/ActiveLogin.Authentication.BankId.AspNetCore/Models/BankIdQrCodeGenerator.cs

```diff
@@ -0,0 +1,19 @@
+using QRCoder;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Models
+{
+    public class BankIdQrCodeGenerator : IBankIdQrCodeGenerator
+    {
+        public string GenerateQrCodeAsBase64(string autoStartToken)
+        {
+            QRCodeGenerator qrGenerator = new QRCodeGenerator();
+
+            var qrUrl = $"bankid:///?autostarttoken={autoStartToken}";
```

Comment:
Please use the `QueryBuilder` as we used to compose theurl in `BankIdLauncher`. This ensures we do encoding correct etc.  But as you have done, the only param we have is `autostarttoken`. So just the query building part.

Reply:
Will do!